server/helpers/passwordHandler.js

```diff
@@ -6,6 +6,15 @@ import bcrypt from 'bcrypt';
  * and the second to retrieve hashed password
  */
 class Hasher {
+  /**
+     * Hashs the password.
```

Comment:
Change the message to **This following method hashes a password**

Reply:
Feedback implemented